app/server/mocks/prisma/singleton.ts

```diff
@@ -0,0 +1,18 @@
+// From prisma docs https://www.prisma.io/docs/guides/testing/unit-testing#singleton
+import { PrismaClient } from '../../src/__generated__/prisma';
+import { mockDeep, mockReset } from 'jest-mock-extended';
+import { DeepMockProxy } from 'jest-mock-extended/lib/cjs/Mock';
+
+import prisma from './client';
```

Comment:
> Thank you for continually iterating on this!
> 
> I have one general question: What's the motivation behind calling the resolver tests integration tests and the method tests unit tests?

I like having the ability to run them separately and using the naming convention is a simple way to do that as I can just filter on the file name pattern. Since the integration tests rely on starting up the server and test DB they take longer to run so it is nice to have the option to run the unit tests separately when working on them.

Reply:
Oh, okay if it's causing more issues than just the import then just forget about it.  Mocking has always been kinda hacky and it's not worth putting in too much effort for the import here.  Just maybe make a note of this error in a comment somewhere.